main.js

```diff
@@ -10,6 +12,40 @@ function changeDirection(event) {
   } else if (event.key === 'ArrowRight') {
     $raceCar.className = 'race-car right';
   }
+  if (event.keyCode === 32) {
+    setInterval(goButton, 16);
+  }
 }
 
+function goButton(event) {
+  // if (event.keyCode === 32) {
+  if ($raceCar.className === 'race-car right') {
+    carLeft += 5;
+    $raceCar.style.left = carLeft + 'px';
+    if (carLeft >= 900) {
+      carLeft -= 5;
+    }
```

Comment:
Is this for making the car not go past the edges? If so, updating `carLeft` won't undo the change you already did to the left property. I would change the work for this conditional block to something like this:

```
  if ($raceCar.className === 'race-car right') {
    if (carLeft < 900) {
      carLeft += 5;
      $raceCar.style.left = carLeft + 'px';
    }
}
```